fable-test-app/src/pages/SubmitHoliday.tsx

```diff
@@ -1,16 +1,111 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 
 // Components (internal)
-import { DateModified, Heading, Text } from '../components';
+import {
+  DateModified,
+  Heading,
+  Text,
+  StepOne,
+  StepTwo,
+  Success
+} from '../components';
 
 const SubmitHoliday: React.FC = () => {
+  const [step, setStep] = useState(1);
+  const [formData, setFormData] = useState({
+    holidayname: '',
+    newholiday: '',
+    holidaydate: '',
+    learnofholiday: '',
+    holidaytype: {
+      federal: false,
+      national: false,
+      other: false
+    },
+    otherholiday: '',
+    province: '',
+    image: null,
+    fullname: '',
+    email: ''
```

Comment:
Can we use camelCase here to make it a bit easier to read? Especially for three-word variables like `learnOfHoliday`